catalogue_of_life/Loader.java

```diff
@@ -0,0 +1,242 @@
+package com.vaticle.typedb.example.catalogueOfLife;
+
+import cli.LoadOptions;
+import com.univocity.parsers.common.record.Record;
+import com.univocity.parsers.tsv.TsvParser;
+import com.univocity.parsers.tsv.TsvParserSettings;
+import com.univocity.parsers.tsv.TsvWriter;
+import com.univocity.parsers.tsv.TsvWriterSettings;
+import loader.TypeDBLoader;
+import picocli.CommandLine;
+
+import java.io.*;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+public class Loader {
+    public static void main(String[] args) throws IOException {
+        Optional<Options> maybeOptions = parseCLIOptions(args);
+        if (maybeOptions.isEmpty()) return;
```

Comment:
Usage, version, or error

Reply:
`maybe` here is to distinguish from bare `Options options` two lines down. I'd rather not have `options.get()` all over the place.

Reply:
ok i got it - then i'd do `parsedOptions` or `optionsOpt` (existing style in other places for encoding the optionality)